### DISCUSSION TITLE
Inject Variables to Another Module

### DISCUSSION THREAD

**Discussion by oribs1** [cue-lang/cue#1530] [Q&A] [answered]
Is it possible to inject a variable into another module?
I have module A with the following definition:
```
env: string @tag(env)
```
In module B I'm importing this module, and I want to be able to inject this variable from the command line:
```
cue eval demo.cue -t env="prod"
```
This is working great when I'm in the same package, but when the tag is in a different module I get:
```
no tag for "env"
```

**Comment by verdverm** [comment, marked as answer]
This is not possible.

- tags are only supported on fields
- field attributes do not carry through imports or unification, tags are an attribute known by the `cue` cli

If this was allowed, you would see problems with tag collisions that you could not control. Imagine not being able to import two packages because they define the same tag, both of which are out of your control.

Or we would have to namespace them which could make using them at the command line more tedious.
